ia/src/parser.py

```
@@ -3,7 +3,7 @@
 from utils import exit_error
 from models import iaParser
 
-def ia_parser(argv: list[str]) -> iaParser:
+def ia_parser(argv: list[str]) -> iaParser or None:
```

Ruff (0.11.9)

`ia_parser` is too complex (11 > 10)

(C901)